### ISSUE TITLE
Make list of items in InfoStation scrollable

### ISSUE THREAD

**Issue by David-S-Morgan** [Cosmic-Chatter/Constellation#2]
Adding more than about 10 items in an InfoStation instance results in additional items being hidden.  Ideally the list of items should be scrollable, in the same way that the text of each item already is if it goes over a page.

We have a project where we have many more than 10 items, and this is a showstopper for us.

**Comment by David-S-Morgan** [comment]
Actually, seems like an easy fix.

```
--- infostation.css_ORIG        2024-04-22 12:11:58.222395113 -0400
+++ infostation.css     2024-04-22 12:12:01.914452375 -0400
@@ -201,4 +201,5 @@
 #buttonRow {
   height: calc(6vmax * var(--button-rows));
+  overflow-y: scroll;
 }
```
 I am not a web developer -- just figured this out from some creative searching. :-)

**Comment by forceflow1049** [comment]
Hi @David-S-Morgan, thanks for another good bug report! None of our uses so far have had enough tabs to trigger this situation (seems to be 10 tabs in the default text size).

InfoStation needs to handle both the cases of a horizontal orientation (where your solution is right!) and vertical orientation (where tabs could stack into multiple rows or scroll horizontally). But that doesn't seem tricky.

I've fleshed this out in a new Issue on the Exhibitera repo (where we have moved development to):
https://github.com/Cosmic-Chatter/Exhibitera/issues/1
